graph-metrics-exporter/overlays/aws-prod/imagestreamtag.yaml

```diff
@@ -7,5 +7,5 @@ spec:
     - name: latest
       from:
         kind: DockerImage
-        name: quay.io/thoth-station/graph-metrics-exporter:v0.6.0
+        name: quay.io/thoth-station/graph-metrics-exporter:v0.6.1
```

Comment:
Is the prod bump expected?

Reply:
as patch that is not braking anything, I thought we could have it also in prod directly, but I can remove aws, wdyt?

Reply:
sure we can do it , if we are sure on that it wont break.